todo.js

```diff
@@ -27,8 +27,14 @@ function renderTodoList() {
 
   todos.forEach((todo) => {
     const newTodoItem = document.createElement("li");
+    newTodoItem.textContent = todo.value;
 
-    if (todo.editState) {
+    if (todo.doneState) {
```

Comment:
Let's break down this `if..else-if..else` into functions. 
So generally speaking the code will look something like this:
```js
const newTodoItem = null
if (todo.doneState) {
   newTodoItem = manageDoneState()
} else if (todo.editState) {
   newTodoItem = manageEditState()
} else {
  newTodoItem = manageInitialState()
}
tasks.appendChild(newTodoItem);
```